src/main/java/com/refinedmods/refinedstorage/RSItems.java

```diff
@@ -23,7 +23,7 @@
 import java.util.Map;
 
 public final class RSItems {
-    private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, RS.ID);
+    public static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, RS.ID);
```

Comment:
Maybe we can move `DeferredRegister` to the `ColorMap` constructor so we can retain private?